features/crud.feature

```diff
@@ -32,8 +34,8 @@ Feature: Create-Retrieve-Update-Delete
       "dummy": null,
       "dummyBoolean": null,
       "dummyDate": "2015-03-01T10:00:00+00:00",
-      "dummyFloat": null,
-      "dummyPrice": null,
+      "dummyFloat": 1.2,
+      "dummyPrice": "1.2",
```

Comment:
This is no longer failing (see [Travis build](https://travis-ci.org/api-platform/core/jobs/165473537)). So the bug is no longer present.